components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/listener/IdentityStoreEventListener.java

```diff
@@ -62,6 +62,7 @@ public class IdentityStoreEventListener extends AbstractIdentityUserOperationEve
     private boolean isHybridDataStoreEnable = false;
     private static final String INVALID_OPERATION = "InvalidOperation";
     private static final String USER_IDENTITY_CLAIMS = "UserIdentityClaims";
+    private static final String STORE_IDENTITY_CLAIMS = "StoreIdentityClaims";
```

Comment:
I believe this needs to be a public constant. It needs to be added to the userstore OOTB for any userstore we create in the onprem.

Reply:
Addresed with https://github.com/wso2-extensions/identity-governance/pull/648/commits/be21dd245b9ea0166557cb6fedd75f93c49ff516